e2e/test/scenarios/models/reproductions.cy.spec.ts

```diff
@@ -980,7 +980,7 @@ describe("issue 34574", () => {
     createQuestion(modelDetails).then(({ body: { id: modelId } }) =>
       visitModel(modelId),
     );
-    cy.wait(["@rootCollection", "@card", "@metadata", "@dataset"]);
```

Comment:
@kamilmielnik added this in #49382, but the test is now failing. It only passed when I remove this and [this](https://github.com/metabase/metabase/pull/49545/files#diff-1ca0eb3ebbd8d8ebc3f1b156b2498e59ee24b0d260dbcb5cb6de4613119e0e2aL997) intercept.